src/app/components/admin-app/admin-view-search/admin-view-edit/admin-view-edit.component.ts

```diff
@@ -244,6 +246,13 @@ export class AdminViewEditComponent implements OnInit {
    * Saves the current view
    */
   saveView(): void {
+
+    if (this.viewNameFormControl.hasError || this.descriptionFormControl.hasError) {
```

Comment:
Suggestion:

`this.hasErrors = this.viewNameFormControl.hasError || this.descriptionFormControl.hasError`